docs/Windows-WSL.md

```diff
@@ -57,6 +71,20 @@ sudo make install
 
 This will install the SDL2 development headers and libraries into `/usr/local/cross-tools/x86_64-w64-mingw32/` if you use a different directory then you will have to supply the SDL2 dir to the `cmake` command below using `-DSDL2_DIR=/usr/local/cross-tools/x86_64-w64-mingw32/lib/cmake/SDL2`
 
+### Installing SLD2-Image
+
+SDL2 Image can be a little trickier to build from source, so we'll grab the pre-compiled mingw packages and install them alongside SDL2 in `/usr/local/cross-tools/x86_64-w64-mingw32/`:
+
+```shell
+wget https://www.libsdl.org/projects/SDL_image/release/SDL2_image-devel-2.0.5-mingw.tar.gz
+tar xzf SDL2_image-devel-2.0.5-mingw.tar.gz
+sudo cp -r bin /usr/local/cross-tools/x86_64-w64-mingw32/
```

Comment:
`sudo cp -r x86_64-w64-mingw32 /usr/local/cross-tools/` ?

Reply:
Ah you're right- I saw the additional files in the root directory and switched- unecessarily- to copying only the bits I wanted. But they're in a subdirectory anyway.